### DISCUSSION TITLE
Account being deleted, transfer repo to another user

### DISCUSSION THREAD

**Discussion by adridos** [community/community#30563] [Other Feature Feedback, Questions, & Ideas]
I need to transfer a repository to another user within my organization but there are a few factor I need to consider when I decide how to do this.

1. My account will be deleted because my internship with this company ends next week
2. The repo I want to transfer was generated from another repo that was forked from a repository under a co-workers account

Can anyone advise me towards the best way to transfer the repo to a coworker?

**Comment by ichenglin** [comment]
For **transferring a repository** hosted under a **personal account**,
simply **follow** the **[official step-by-step guide](https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository#transferring-a-repository-owned-by-your-personal-account)** to do so.

There **shouldn't be any problem** with transferring a **forked repository**,
unless the one you're trying to **transfer the repository** to **already owns a fork** or it was a **private fork**.

> **Prerequisites for repository transfers** **[(documentation)](https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository#about-repository-transfers)**
>
> - When you transfer a repository that you own to another personal account, the new owner will receive a confirmation email. The confirmation email includes instructions for accepting the transfer. If the new owner doesn't accept the transfer within one day, the invitation will expire.
> - To transfer a repository that you own to an organization, you must have permission to create a repository in the target organization.
> - The target account **`must not have a repository with the same name, or a fork in the same network`**.
> - The original owner of the repository is added as a collaborator on the transferred repository. Other collaborators to the transferred repository remain intact.
> - **`Private forks`** can't be transferred.

The **following picture** shows the **transfer repository button**,
which was located at **`Repository > Settings > Danger Zone > Transfer ownership`**.

Transfer Repository Guide: **[(documentation)](https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository#transferring-a-repository-owned-by-your-personal-account)**

![image](https://user-images.githubusercontent.com/41904540/186221073-fd533608-6e88-4286-83df-712d46ea3766.png)

**Reply by adridos** [reply]
Thank you for the response!

**Comment by Nateay0** [comment]
You have to use this
![image](https://user-images.githubusercontent.com/92893096/186284760-22493215-4f10-489e-816e-5301af1f0f81.png)
